docs/modules/ROOT/pages/configuration-store.adoc

```diff
@@ -0,0 +1,172 @@
+= Universal Configuration Store
+
+== Overview
+The Universal Configuration Store is a tool that enables the various configurations for a project to be centrally defined and managed. 
+It then provides a standardized way of accessing them, allowing the environment specific configurations to be dynamically provided to 
+their respective resources within the project at runtime.
+
+=== Setup
+To setup the configuration store, one must first create properties files for storing the configurations. Each individual configuration is 
+represented as a property key value pair within the file. These files must then be made accessible to the configuration store kubernetes 
+resource for ingestion. See the applicable sections below for more details on defining configurations as properties, creating environment 
+specific property overrides, mounting the properties, and specifying the configuration store backend storage type.
+
+==== Defining Configuration Properties
+Properties represent key values pairs for storing the project configurations. The key values pairs are stored as string types using 
+the standard `.properties` file format. Each property then has an associated group name determined by the name of the file. Properties
+are then accessed using a combination of their group name and property name. Accessing the properties is covered in more detail in the 
+<<Usage>> section below.
+
+Here is an example of a base properties file named `messaging.properties`:
+[source,properties]
+----
+connector=smallrye-kafka
+topic=baseTopic
+----
+This file defines two properties, `connector` and `topic`, each with the group name `messaging`.
+
+===== Environment Specific Overrides
+In addition to the base properties file introduced above, one can optionally specify a secondary properties file that houses environment
```

Comment:
S: Again I think we can largely lean on Krausening docs here and only call out the aiSSEMBLE-specific stuff.

Reply:
Looking at the [krausening related section for this](https://github.com/TechnologyBrewery/krausening/tree/dev/krausening/), while it does explain how the extension works, I think it introduces too many unrelated details. Specifically the stuff the env var `KRAUSENING_EXTENSIONS` and the code block for retrieving the properties.

Would rather just keep a straightforward example here that just focus on the properties files.